packages/gestalt/src/WashAnimated.js

```diff
@@ -22,21 +22,27 @@ type Props = {|
    */
   image?: Node,
   /**
-   * Optional callback fired when the user mouses over Card.
+   * Optional callback fired when the user mouses over WashAnimated.
    */
   onMouseEnter?: ({| event: SyntheticMouseEvent<HTMLDivElement> |}) => void,
   /**
-   * Optional callback fired when the user away from Card.
+   * Optional callback fired when the user away from WashAnimated.
```

Comment:
looks like this has had a typo: "…when the user _mouses_ away from…"

Reply:
done